club/theme.json

```diff
@@ -46,6 +46,16 @@
 					"slug": "background",
 					"color": "#F6EB80",
 					"name": "Background"
+				},
+                {
+					"slug": "primary",
+					"color": "#000AFF",
+					"name": "Primary"
+				},
+				{
+					"slug": "tetriary",
```

Comment:
Kind of a nitpick, but `tertiary` is technically third order I believe, so should this actually be `secondary`?

Reply:
:thinking: I saw this in many of our latest themes. Tertiary color without a secondary one. I think the name can be related to the color theory and not to the number of colors in our palette. Primary is, of course, the primary color, and background would be the secondary.

Reply:
I think that made more sense when we used the customizer for colors, but I don't know that it makes sense in block themes.